packages/x-components/src/components/location-injector.vue

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import { Prop, Provide, Component } from 'vue-property-decorator';
+  import { NoElement } from '../components/no-element';
+  import { FeatureLocation } from '../types';
+
+  /**
+   * Location Injector component.
+   * This component injects the location with value passed as prop.
+   *
+   * @public
+   */
+  @Component
+  export default class LocationInjector extends NoElement {
+    /**
+     * {@link FeatureLocation | Location} To inject.
```

Comment:
Do not rename the type if not necessary. Also if you add a *The* you don't have to wrongly capitalise the *To*.
```suggestion
      * The {@link FeatureLocation} to inject.
```